src/containers/walletConnect/index.tsx

```diff
@@ -117,7 +118,9 @@ const WalletConnect: React.FC<WalletConnectProps> = ({actionIndex}) => {
   ) {
     return (
       <ModalBottomSheetSwitcher isOpen={true}>
-        <div className="h-40" />
+        <div className="h-40">
```

Comment:
Changing the div styling makes it symmetrical
```suggestion
        <div className="pb-36">
```